README.md

```diff
@@ -42,8 +42,7 @@ match '/saml/logout' => 'saml_idp#logout', via: [:get, :post, :delete]
 Create a controller that looks like this, customize to your own situation:
 
 ``` ruby
-class SamlIdpController
-  include SamlIdp::IdpController
+class SamlIdpController < SamlIdp::IdpController
```

Comment:
`SamlIdp::Controller` is a Concern, not a Class, and should be `include`d not subclassed.

`SamlIdp::IdpController` is in `app/controller` and is a Class. It is not referred to in the README afaik.